src/main/java/uk/gov/hmcts/reform/ccd/document/am/service/DocumentManagementService.java

```diff
@@ -28,5 +27,5 @@
 
     ResponseEntity patchDocument(final UUID documentId, UpdateDocumentCommand updateDocumentCommand);
 
-    boolean patchDocumentMetadata(DocumentMetadata caseDocumentMetadata);
```

Comment:
Need to understand how this api is returning boolean response now?

Reply:
There is no requirement for boolean response anymore, It will simply response 200 with success string and 400 series for negative response.